server/src/test/java/io/spine/system/server/given/AbstractEventAccumulator.java

```diff
@@ -102,24 +103,26 @@ public void assertEventCount(int expectedCount) {
 
     public void forgetEvents() {
         events.clear();
-        eventIterator = null;
+        nonCheckedEvents.clear();
     }
 
     @CanIgnoreReturnValue
-    public <E extends Message> E assertNextEventIs(Class<E> eventType) {
-        if (eventIterator == null) {
-            eventIterator = ImmutableList.copyOf(events).iterator();
-        }
-        assertTrue(eventIterator.hasNext(), errorMessage());
-        Message next = eventIterator.next();
-        assertThat(next, instanceOf(eventType));
+    public <E extends Message> E assertExistEvent(Class<E> eventType) {
```

Comment:
The naming is confusing from both grammar and contextual point of view.

As there are no docs for the method, I am not able to suggest a name.